pkg/kn/commands/types.go

```diff
@@ -94,6 +101,16 @@ func (params *KnParams) newSourcesClient(namespace string) (sources_kn_v1alpha1.
 	return sources_kn_v1alpha1.NewKnSourcesClient(client, namespace), nil
 }
 
+func (params *KnParams) newEventingClient(namespace string) (eventing_kn_v1alpha1.KnEventingClient, error) {
+	restConfig, err := params.prepareRestConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	client, _ := eventing.NewForConfig(restConfig)
```

Comment:
Why are we ignoring errors here?

Reply:
Errors are ignored here in the similar methods, e.g. `newServingClient`. I keep same to be consistent. We could update them all in consistent in separate PR.